### PR TITLE
JA: add site/ja/tutorials/text/text_generation.ipynb

### PR DESCRIPTION
**Pull request by masa-ita**

### CODE REVIEW

File: site/ja/tutorials/text/text_generation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1256 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BwpJ5IffzRG6"
+      },
+      "source": [
+        "このチュートリアルでは、文字ベースの RNN を使ってテキストを生成する方法を示します。ここでは、Andrej Karpathy の [The Unreasonable Effectiveness of Recurrent Neural Networks](http://karpathy.github.io/2015/05/21/rnn-effectiveness/) からのシェイクスピア作品のデータセットを使います。このデータからの文字列（\"Shakespear\"）を入力にして、文字列中の次の文字（\"e\"）を予測するモデルを訓練します。このモデルを繰り返し呼び出すことで、より長い文字列を生成することができます。\n",
```
Comment (yukinagae) [inline site/ja/tutorials/text/text_generation.ipynb]: [suggestion] Maybe the below translation seems to be better to me if I were to translate the sentence `Given a sequence of characters from this data ("Shakespear")`.

`このデータからの文字列（\"Shakespear\"）を入力にして` -> `このデータ（\"Shakespear\"）からの文字列を入力にして`

File: site/ja/tutorials/text/text_generation.ipynb [hunk 2]
```diff
@@ -0,0 +1,1256 @@
...
+        "\n",
+        "PETRUCHIO:\n",
+        "I say he look'd on, if I must be content\n",
+        "To stay him from the fatal of our country's bliss.\n",
+        "His lordship pluck'd from this sentence then for prey,\n",
+        "And then let us twain, being the moon,\n",
+        "were she such a case as fills m\n",
+        "</pre>\n",
+        "\n",
+        "いくつかは文法にあったものがある一方で、ほとんどは意味をなしていません。このモデルは、単語の意味を学習していませんが、次のことを考えてみてください。\n",
```
Comment (yukinagae) [inline site/ja/tutorials/text/text_generation.ipynb]: This paragraph is nice :)

File: site/ja/tutorials/text/text_generation.ipynb [hunk 3]
```diff
@@ -0,0 +1,1256 @@
...
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "aavnuByVymwK",
+        "colab": {}
+      },
+      "source": [
+        "# 読み込んだのち、Python 2 との互換性のためにデコード\n",
```
Comment (yukinagae) [inline site/ja/tutorials/text/text_generation.ipynb]: This translation is pretty good!

- Original: `Read, then decode for py2 compat.` <- unnecessasry abbreviation...

File: site/ja/tutorials/text/text_generation.ipynb [hunk 4]
```diff
@@ -0,0 +1,1256 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "bbmsf23Bymwe"
+      },
+      "source": [
+        "### 推論タスク"
```
Comment (yukinagae) [inline site/ja/tutorials/text/text_generation.ipynb]: [weak suggestion] `推論タスク` -> `予測タスク`

File: site/ja/tutorials/text/text_generation.ipynb [hunk 5]
```diff
@@ -0,0 +1,1256 @@
...
+        "colab_type": "code",
+        "id": "p2pGotuNzf-S",
+        "colab": {}
+      },
+      "source": [
+        "# バッチサイズ\n",
+        "BATCH_SIZE = 64\n",
+        "\n",
+        "# データセットをシャッフルするためのバッファサイズ\n",
+        "# （TD data は可能性として無限長のシーケンスでも使えるように設計されています。\n",
```
Comment (yukinagae) [inline site/ja/tutorials/text/text_generation.ipynb]: [typo] `TD` -> `TF`

File: site/ja/tutorials/text/text_generation.ipynb [hunk 6]
```diff
@@ -0,0 +1,1256 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "IxdOA-rgyGvs"
+      },
+      "source": [
+        "訓練時間を適切に保つために、10エポックを使用してモデルを訓練します。"
```
Comment (yukinagae) [inline site/ja/tutorials/text/text_generation.ipynb]: [suggestion] missing one sentence?

`In Colab, set the runtime to GPU for faster training.`

File: site/ja/tutorials/text/text_generation.ipynb (hunk 1 above)
Comment (masa-ita) [reply site/ja/tutorials/text/text_generation.ipynb]: I think the string "Shakespear" is a part of "Shakespeare" to feed the model and predict the last "e" .

File: site/ja/tutorials/text/text_generation.ipynb (hunk 2 above)
Comment (masa-ita) [reply site/ja/tutorials/text/text_generation.ipynb]: Thanks!

File: site/ja/tutorials/text/text_generation.ipynb (hunk 3 above)
Comment (masa-ita) [reply site/ja/tutorials/text/text_generation.ipynb]: Thank you.

File: site/ja/tutorials/text/text_generation.ipynb (hunk 4 above)
Comment (masa-ita) [reply site/ja/tutorials/text/text_generation.ipynb]: Thank you.  Fixed.

File: site/ja/tutorials/text/text_generation.ipynb (hunk 5 above)
Comment (masa-ita) [reply site/ja/tutorials/text/text_generation.ipynb]: Oops.  Fixed.

File: site/ja/tutorials/text/text_generation.ipynb (hunk 6 above)
Comment (masa-ita) [reply site/ja/tutorials/text/text_generation.ipynb]: Thanks.  Added "Google Colab を使用する場合には、訓練を高速化するためにランタイムを GPU に設定します。".

File: site/ja/tutorials/text/text_generation.ipynb (hunk 1 above)
Comment (yukinagae) [reply site/ja/tutorials/text/text_generation.ipynb]: Oh, I got your point now! You're right :)

**Pull request merged**